macros/contextFraction.pl

```diff
@@ -907,11 +907,16 @@ sub string {
 sub TeX {
   my $self = shift; my $equation = shift; shift; shift; my $prec = shift;
   my ($a,$b) = @{$self->{data}}; my $n = "";
+  my $textstyle = '';
   return "$a" if $b == 1;
-  if ($self->getFlagWithAlias("showMixedNumbers","showProperFractions") && CORE::abs($a) > $b)
-    {$n = int($a/$b); $a = CORE::abs($a) % $b; $n .= " " unless $a == 0}
+  if ($self->getFlagWithAlias("showMixedNumbers","showProperFractions") && CORE::abs($a) > $b) {
+	$n = int($a/$b);
+	$a = CORE::abs($a) % $b;
+	$n .= ' ' unless $a == 0;
+	$textstyle = '\\textstyle';
```

Comment:
The indentation of these lines is inconsistent with the rest of the file.  This file uses a 2 space indentation scheme, and these lines are indented with a single tab.

Unfortunately, this file (and most of PG) uses an indentation scheme that is inconsistent with what I usually promote.  That being to use tab indentation instead of spaces.  Tab indentation does not force the indentation amount on whoever is viewing the file.  Spaces do.

In any case, mixing tab and space indentation is even more troublesome.  If someone has their tabstop set to anything other than 4, then this will not line up correctly in the scope of the space indented file.